config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/TrustedCertLoader.java

```diff
@@ -33,6 +37,11 @@ public void load(Bundle bundle, Element element) {
         TrustedCert cert = new TrustedCert(properties, getCertData(trustedCertElem));
         cert.setId(trustedCertElem.getAttribute(ATTRIBUTE_ID));
         cert.setName(getSingleChildElementTextContent(trustedCertElem, NAME));
+        Set<Annotation> annotations = new HashSet<>();
```

Comment:
if we are storing id in yml file, we cant we directly store instead of bundle-entity annotation

Reply:
@uppoju , No, that is against DevPlugin policy :)